CODEOWNERS

```diff
@@ -26,6 +26,7 @@ packages/repository/* @raymondfeng
 packages/repository-tests/* @bajtos
 packages/repository-json-schema/* @bajtos
 packages/rest/* @bajtos @raymondfeng
+packages/rest-crud/* @bajtos @hacksparrow
```

Comment:
Discussion point: should this new package live in `packages/rest-crud` or `extensions/rest-crud`?

Reply:
My personal take: `packages/rest-crud`.